### ISSUE TITLE
mixin ignores mixed-in class type

### ISSUE THREAD

**Issue by idoros** [wix/stylable#121]
```css
@namespace "comp";
:import {
    -st-from: "./button.st.css";
    -st-default: Button;
}
.clicker {
    -st-extends: Button;
}
.clicker::label {
    color:red;
}
.myBtn {
    -st-mixin: clicker;
}
```
Will output:
```css
...
.comp--root .comp--myBtn::label { color:red; }
```
Should output:
```css
...
.comp--root .comp--myBtn.button--root .button--label { color:red; }
```

**Comment by tomrav** [comment]
Was able to reproduce this - the issue still occurs.

**Comment by tomrav** [comment]
Resolved in v5.1.0 
https://github.com/wix/stylable/releases/tag/v5.1.0
